### PR TITLE
[full-ci] [tests-only] Added tests for PROPFIND as sharee for folder

### PR DESCRIPTION
**Pull request by grgprarup**
## Description
As sharee, PROPFIND the shares, when two users share resources with the same name using graph API.

Restructured the scenario so that it will cover the folder as well.

## Related Issue

- Part of https://github.com/owncloud/ocis/issues/8493
- Coverage for https://github.com/owncloud/ocis/issues/4421

## How Has This Been Tested?

- Locally
- :robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/apiSharingNg/propfindShares.feature [hunk 1]
```diff
@@ -43,3 +43,33 @@ Feature: propfind a shares
       | oc:permissions   | SR               |
       | oc:size          | 8                |
       | d:getcontenttype | text/plain       |
+
+  @issue-4421
+  Scenario: sharee PROPFIND a shares when multiple users share folder with same name
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/propfindShares.feature:48]: Is it possible to put this and above scenario in example table?

Comment (saw-jan) [reply tests/acceptance/features/apiSharingNg/propfindShares.feature:48]: are they necessary (related to the bug report)?

Comment (grgprarup) [reply tests/acceptance/features/apiSharingNg/propfindShares.feature:48]: > are they necessary (related to the bug report)?

I don't think so. I will refactor with an example table.

File: tests/acceptance/features/apiSharingNg/propfindShares.feature [hunk 2]
```diff
@@ -11,35 +11,37 @@ Feature: propfind a shares
       | Carol    |
 
   @issue-4421
-  Scenario: sharee PROPFIND a shares when multiple user shares resources with same name
+  Scenario Outline: sharee PROPFIND a shares when multiple users share resource with same name
```
Comment (saw-jan) [inline tests/acceptance/features/apiSharingNg/propfindShares.feature:14]:
```suggestion
  Scenario Outline: sharee PROPFIND same name shares shared by multiple users
```

**Review by saw-jan — APPROVED** [review]
:+1:

**Pull request merged**